src/pc/stubs.c

```diff
@@ -119,6 +119,9 @@ s32 D_800987B4;
 u8 g_PadsRepeatTimer[BUTTON_COUNT * PAD_COUNT];
 s32 D_80136410;
 s32 D_80136414[];
+s32 D_8013B65C;
+s32 D_800BD228;
```

Comment:
```
g_XaMusicConfigs = 0x800BD224;
```
So use unk228 for D_800BD228 and  240 is probably the next element, by my count the struct is 16 bytes. So 0x800BD224+16+12 == g_XaMusicConfigs[pos + 1].unk22f?

Reply:
What I needed to do was to add 1 to the position to finally get it to match with the field `unk230` from that same structure, thanks !